Example/Podfile.lock

```diff
@@ -16,7 +16,7 @@ PODS:
   - MoEngagePluginInbox (1.0.0):
     - MoEngageInbox (< 1.4.0, >= 1.3.0)
     - MoEngagePluginBase (< 3.1.0, >= 3.0.0)
-  - MoEngageRichNotification (6.3.1):
+  - MoEngageRichNotification (6.3.2):
```

Comment:
@RakshithaAcharya Any reason for committing podfile.lock changes?

Reply:
@deepa-iOS-SDK This is part of sample app. This Keeps the track of version in which last testing was done.